library/general/src/lib/ui/event_dispatcher.rb

```diff
@@ -0,0 +1,65 @@
+require "yast"
+
+module UI
+  # Provides switch between event_loop and dispatching to handlers.
+  # @example simple OK/cancel dialog
+  #   class OKDialog
+  #     include Yast::UIShortcuts
+  #     include Yast::Logger
+  #     include UI::EventDispatcher
+  #     Yast.import "UI"
+  #
+  #     def run
+  #       return nil unless Yast::UI.OpenDialog(
+  #         HBox(
+  #           PushButton(Id(:ok), "OK"),
+  #           PushButton(Id(:cancel), "Cancel")
+  #         )
+  #       )
+  #       begin
+  #         return event_loop
+  #       ensure
+  #          Yast::UI.CloseDialog
+  #       end
+  #     end
+  #
+  #     def ok_handler
+  #       finish_dialog(:ok)
+  #       log.info "OK button pressed"
+  #     end
+  #   end
+  module EventDispatcher
+    # @internal constant to continue with dispatching
+    CONTINUE_WITH_DISPATCHING = :_dispatcher_continue
```

Comment:
Why not using the symbol directly? I don't find much added value in defining a constant.